testgrid/config/README.md

```diff
@@ -13,11 +13,32 @@ User should only add/update config.yaml
 -- Yaml representation for configuring testgrid.k8s.io
 
 To add a new test:
+
 1. Append a new testgroup under test_groups, specity the name and where to get the log.
+
 2. Append a new dashboardtab under the dashboard you would like to add the testgroup to,
+
   or create a new dashboard and assign the testgroup to the dashboard.
+  
   * The testgroup name from a dashboardtab should match the name from a testgroup
+  
   ** Note that a testgroup can be within multiple dashboards. 
-3. Run config_test.go to make sure the config is valid.
+  
+3. Run `config_test.go` to make sure the config is valid.
+
 
 Contact us before you make changes to config.proto
+
+#
+Devs -
+If you changed config.proto, do:
+
+1. Make sure [protoc](https://github.com/golang/protobuf) is installed and
+
```

Comment:
FYI, I don't think you need two LB here, but it is needed between `do:` and `1.`